1.9/deploying-services/pods/technical-overview.md

```diff
@@ -151,6 +151,7 @@ The following JSON specifies a Docker image for a pod:
 
 # Limitations
 
+- Pods are not supported in DC/OS Enterprise `strict` security mode.
```

Comment:
`Enterprise DC/OS`, not the other way around. Also, no need to have strict in command syntax - it would probably be more useful if you made it all a hyperlink to the definition instead, e.g.: `Enterprise DC/OS [strict security mode](docs.mesosphere.com/security/..).` .